### PR TITLE
Accessibility Dropdown Fixes

### PR DESCRIPTION
**Pull request by emichaud998**
Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

Changes in this PR:
- Moved the "Select All" and "Deselect All" buttons outside of the FHIR version dropdown since it was not possible to tab to the buttons within the dropdown
- Changed colors of dropdowns and buttons to make them stand out more
- Change the aria labels of all of the dropdowns to better explain how to use and navigate them

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: [Lantern 518 JIRA Ticket](https://oncprojectracking.healthit.gov/support/browse/LANTERN-518)
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge
is complete.

### CODE REVIEW

File: shinydashboard/lantern/ui.R [hunk 1]
```diff
@@ -379,17 +412,24 @@ ui <- dashboardPage(
         for (let mutation of mutations) {
           if (mutation.addedNodes.length > 0) {
             for (let newNode of mutation.addedNodes) {
+              
+              if (mutation.target.id === \"show_date_filters\" && newNode.classList && newNode.classList.contains(\"row\")) {
+                let selectInputNodes = newNode.querySelectorAll(\"select.shiny-bound-input\")
+                for (let selectInputNode of selectInputNodes) {
+                  selectInputNode.setAttribute('aria-label', 'Use the arrow keys to naviate the filter menu.')
```
Comment (bmath10) [inline shinydashboard/lantern/ui.R:419]: misspelling: "navigate"

File: shinydashboard/lantern/ui.R [hunk 2]
```diff
@@ -404,7 +444,29 @@ ui <- dashboardPage(
...
+                let ulFieldList = fieldList.getElementsByTagName(\"ul\")[0];
+                ulFieldList.removeAttribute(\"tabindex\");
+              }   
             }
+
+            if (mutation.addedNodes[0].classList && mutation.addedNodes[0].classList.contains(\"container-fluid\")) {
+              let containerNode = mutation.addedNodes[0]
+              let selectDropdowns = containerNode.querySelectorAll(\"select.shiny-bound-input\")
+              for (selectDropdown of selectDropdowns) {
+                selectDropdown.setAttribute('aria-label', 'Use the arrow keys to naviate the filter menu.')
```
Comment (bmath10) [inline shinydashboard/lantern/ui.R:467]: same as above: "navigate"

File: shinydashboard/lantern/ui.R [hunk 3]
```diff
@@ -437,8 +499,20 @@ ui <- dashboardPage(
                       attributeFilter: [\"tabindex\"]
                     });
                 }
-              }
+              }            
             }
+            
+            let selectInputButtons = document.querySelectorAll(\"select.shiny-bound-input\")
+            for (let selectInput of selectInputButtons) {
+              selectInput.setAttribute('aria-label', 'Use the arrow keys to naviate the filter menu.')
```
Comment (bmath10) [inline shinydashboard/lantern/ui.R:507]: same as above

File: shinydashboard/lantern/ui.R [hunk 4]
```diff
@@ -437,8 +499,20 @@ ui <- dashboardPage(
...
+            }
+          }
+          
+          if (mutation.target.id === \"show_filters\") {
+            let dropDownButtons = document.getElementsByClassName(\"dropdown-toggle\")
+            for (let dropDownButton of dropDownButtons) {
+              dropDownButton.setAttribute('aria-label', 'Dropdown filter menu button. Press the down arrow key to open the filter menu, use the tab or arrow keys to navigate through options, press enter to select a filter option, and use the escape key to close the filter menu.')
```
Comment (bmath10) [inline shinydashboard/lantern/ui.R:514]: Is it possible to just have this read out when you're on the dropdown without it being open? I tried navigating with VoiceOver and it was kind of a lot to have this read every time you switched options for the dropdown. If this is the only way to do it then I think it's fine, but just wanted to ask and see if there was an alternative!

**Review by bmath10 — CHANGES_REQUESTED** [review]
This also needs a rebase!

**Comment by bmath10** [comment]
Once this is rebased I can merge it in!

**Comment by bmath10** [comment]
This is kind of a weird bug (maybe due to the rebase?) but when I first load the dashboard, the "Skip Past Resources" on the Resources page is just floating above the FHIR Version dropdown buttons. Once I refresh the page, though, it disappears and only shows up where it's expected above the resources select object.

**Comment by emichaud998** [comment]
> This is kind of a weird bug (maybe due to the rebase?) but when I first load the dashboard, the "Skip Past Resources" on the Resources page is just floating above the FHIR Version dropdown buttons. Once I refresh the page, though, it disappears and only shows up where it's expected above the resources select object.

Weird I am not seeing this, I tried it both by booting Lantern up using R shiny and opening it up in the browser from there, and I also tried it by going to localhost:3838, but I am not seeing the Skip to Content floating there

**Pull request merged**